.github/workflows/test_with_clone.yaml

```diff
@@ -0,0 +1,111 @@
+---
+name: Test Changes with Cloned DB
+
+on:
+    pull_request:
+        types: [ labeled, closed ]
+
+permissions:
+    contents: read
+
+jobs:
+    
+    create_clone_and_run_schemachange:
+        runs-on: ubuntu-latest
+        if: contains(github.event.pull_request.labels.*.name, 'create_clone_and_run_schemachange')
```

Comment:
Won't this job still run if the PR is closed and it contains the 'create_clone_and_run_schemachange' label?

Reply:
Good catch! On second thought, I think it would. I've updated the logic so that this job only runs on open PRs: https://github.com/Sage-Bionetworks/snowflake/pull/78/commits/03dfc5ac8ae02b867c78c68f555b9b2e67434b5e